schemas/clab.schema.json

```diff
@@ -62,6 +62,9 @@
                         "vr-vjunosswitch",
                         "vr-juniper_vjunosswitch",
                         "juniper_vjunosswitch",
+                        "vr-vjunosevolved",
+                        "vr-juniper_vjunosevolved",
```

Comment:
safe to remove this. we gradually cleanup vr- prefixed names in favor of <vendor>_<nos> format